operations/boot_orchestration/BOS_API_Versions.md

```diff
@@ -0,0 +1,41 @@
+# Boot Orchestration
+
+The Boot Orchestration Service \(BOS\) currently supports two API versions, v1 and v2, that have different APIs and underlying mechanisms for performing operations on nodes.
+The following is a summary of the changes, and the upgrade path, for users wishing to compare the two.
+
+## BOS v2 improvements
+
+BOS v2 makes significant improvements to boot times, retries, and error handling, by allowing nodes to proceed through the boot process at their own pace.
+Nodes that require extra time or retries to complete will no longer hold up the rest of the nodes, meaning that more nodes will reach a ready state faster.
+
+## API Differences
+
+The v2 endpoints have plural nouns, using `sessions` and `sessiontemplates` rather than `session` and `sessiontemplate`.  This is more consistent with our other APIs.
+
+BOS v2 also introduces two new endpoints.  A `/components` endpoint for tracking state at a components level and an `/options` endpoint to allow changing global options dynamically.
+See [Components](Components.md) and [Options](Options.md) for more information on these new endpoints.
+
+There are some different API responses in BOS v2 as well.  Listing sessions will now result in full session records rather than just a list of the current session names.
+The session status endpoint has also changed significantly.  See [View the Status of a BOS Session](View_the_Status_of_a_BOS_Session.md) for more information on the session status endpoint.
```

Comment:
```suggestion
There are some different API responses in BOS v2 as well: 

- Listing sessions in BOS v2 results in full session records rather than just a list of the current session names.
- The session status endpoint has also changed significantly. See [View the Status of a BOS Session](View_the_Status_of_a_BOS_Session.md) for more information on the session status endpoint.
```